shristy-kumari/Js-task/TIC_TAC_TOE/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tic Tac Toe Game</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="overlay" id="overlay">
+        <div class="overlay-content">
+            <h2 class="welcome-header">Welcome to Tic Tac Toe!</h2>
+            <p class="instruction-header"><strong>Some Important Instrutions before start the game:</strong></p>
```

Comment:
Fix sentence formation, it has grammatical errors.
```suggestion
            <p class="instruction-header"><strong>Some Important Instructions before starting the game:</strong></p>
```